paper/ref.bib

```diff
@@ -112,3 +113,29 @@ @article{ForetsS21
   url       = {https://doi.org/10.21105/jcon.00097},
   doi       = {10.21105/jcon.00097}
 }
+
+@book{neumaier1993wrapping,
+  title={The wrapping effect, ellipsoid arithmetic, stability and confidence regions},
+  author={Neumaier, Arnold},
+  year={1993},
+  publisher={Springer}
+}
+
+@article{karhbet2017range,
+  title={Range bounds of functions over simplices, for branch and bound algorithms},
+  author={Karhbet, Sam and Kearfott, Ralph Baker},
+  journal={Reliable Comput},
```

Comment:
```suggestion
  journal={Reliable Computing},
```

Reply:
Actually, we have another reference from the same journal, so let's go with the same style:
```suggestion
  journal={Reliab. Comput.},
```